MODULE.bazel

```diff
@@ -1,6 +1,147 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_play_routes")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "buildifier_prebuilt", version = "7.3.1")
+bazel_dep(name = "rules_java", version = "7.11.1")
+bazel_dep(name = "rules_jvm_external", version = "6.4")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "rules_scala_annex")
+
+rules_scala_annex_version = "bzlmod-migration"
+
+archive_override(
+    module_name = "rules_scala_annex",
+    integrity = "sha256-Gt+S2R53KPhfsK66bhOSXowjb35Ouf24zcT3xFZcyAs=",
+    strip_prefix = "rules_scala-{}".format(rules_scala_annex_version),
+    urls = ["https://github.com/lucidsoftware/rules_scala/archive/refs/heads/{}.zip".format(rules_scala_annex_version)],
+)
+
+bazel_dep(name = "stardoc", version = "0.7.1")
+
+register_toolchains(
+    "//:repository_default_toolchain_21_definition",
+    "//play-routes-toolchain:play-routes-3",
+    "//play-routes-toolchain:play-routes-2-13",
+    "//scala:zinc_2_13",
+    "//scala:zinc_3",
+)
+
+# Please ensure these stay up-to-date with the versions in `versions.bzl`. Unfortunately,
+# `MODULE.bazel` files can't call `load`, so we have to copy them here.
+play_version = "3.0.4"
+
+# Please ensure these stay up-to-date with the versions in `scala/version.bzl`. Unfortunately,
+# `MODULE.bazel` files can't call `load`, so we have to copy them here.
+scala_2_13_version = "2.13.14"
+
+scala_3_version = "3.5.1"
+
+protobuf_version = "4.28.3"
+
+scopt_version = "4.1.0"
+
+specs2_version = "4.20.8"
+
+zinc_version = "1.10.1"
```

Comment:
Scope creep, but could we update this to 1.10.4 while we're at it? Annex is already on that version, so it shouldn't require anymore work besides just updating the number and generating the lock file.

Reply:
Sure thing. I refrained from updating any versions in the `lucidsoftware/rules_scala_annex` PR because I considered it scope creep, but this is a simple change.

Reply:
Thanks

Reply:
No problem!